chemprop/cli/predict.py

```diff
@@ -274,6 +272,26 @@ def main(args):
     logger.info(f"Predictions saved to '{args.output}'")
 
 
+def main(args):
+    match (args.smiles_columns, args.reaction_columns):
+        case [None, None]:
+            n_components = 1
+        case [_, None]:
+            n_components = len(args.smiles_columns)
+        case [None, _]:
+            n_components = len(args.reaction_columns)
+        case _:
+            n_components = len(args.smiles_columns) + len(args.reaction_columns)
+
+    multicomponent = n_components > 1
+
+    model_paths = find_models(args.model_path)
+
+    for model_path in model_paths:
+        output_dir = args.output.parent / model_path.stem / args.output.stem
```

Comment:
This changes the save location of a model, so the help string for `--output` should also be updated. But also this looses the suffix of `output` which tells us whether to pickle or not. I'd prefer we write to `Path(args.output.stem + f"_{i}" + args.output.suffix)` using `for i, model in enumerate(model_paths)` or something similar so that all prediction files are saved in the same folder. At least for now.